src/config/index.ts

```diff
@@ -4,6 +4,29 @@ import { fileURLToPath } from 'node:url';
 const __filename = fileURLToPath(import.meta.url);
 const __dirname = path.dirname(__filename);
 
+/**
+ * Validates that a required environment variable is set
+ */
+function requireEnv(name: string): string {
+  const value = process.env[name];
+  if (!value) {
+    throw new Error(
+      `FATAL: Required environment variable ${name} is not set. Application cannot start.`
+    );
+  }
+  return value;
+}
+
+/**
+ * Helper to extract base64 secrets
+ */
+function getSecret(envSecret: string): string {
+  if (envSecret.startsWith('base64:')) {
+    return envSecret.substring(7); // Remove 'base64:' prefix
+  }
+  return envSecret;
+}
```

Comment:
what's the origin of this? AFAIK this (the prefix) is normally only used when the values is supposed to be decoded. As this is an encryption key it can be used as is, and I wouldn't expect such strings to be used as env vars on any env